code/ui/components/src/tabs/tabs.helpers.tsx

```diff
@@ -10,25 +12,41 @@ export const VisuallyHidden = styled.div<VisuallyHiddenProps>(({ active }) =>
   active ? { display: 'block' } : { display: 'none' }
 );
 
-export const childrenToList = (children: any, selected: string) =>
+export const childrenToList = (children: TabsProps['children']) =>
```

Comment:
The tabs component is due for a complete overhaul.
The API is has with the children being the interface was not a great idea of mine.

It was hard to get the title to show up and not lose state when the tab became active/inactive.